package-lock.json

```diff
@@ -4,7 +4,6 @@
 	"requires": true,
 	"packages": {
 		"": {
-			"name": "notes",
```

Comment:
Is this intended? If not: please remove this change (this will reduce conflicts)

Reply:
Yeah my IDE always wants to add that. This time it slipped, i will remove it.